api.go

```diff
@@ -1227,6 +1227,143 @@ func (sc *Client) DeleteCorpIntegration(corpName, id string) error {
 	return err
 }
 
+// CloudWAFInstance contains the data for a Cloud WAF Instance read API call.
+type CloudWAFInstance struct {
+	ID                      string                            `json:"id"`
+	Name                    string                            `json:"name"`
+	Description             string                            `json:"description"`
+	Region                  string                            `json:"region"`
+	TLSMinVersion           string                            `json:"tlsMinVersion"`
+	WorkspaceConfigs        []CloudWAFInstanceWorkspaceConfig `json:"workspaceConfigs"`
```

Comment:
replace these fields with CloudWAFInstanceBody so they can share the structure on the client side

Reply:
> maybe additionally check if any of these fields are optional, you might want to put omitEmpty on them. I think you only get in trouble if there are defaults though so it probably isn't an issue. but if a customer leaves it blank expecting a default value and you override it with "" it can cause confusion

The vast majority of the fields for creating and updating Cloud WAF Instances are required.

The required fields are:

1. `workspaceConfig.clientIPHeader`:  The request example shows an empty string rather than missing property so it appears to accepted behavior: https://docs.fastly.com/signalsciences/api/#_corps__corpName__cloudwafInstances_post . Still awaiting information on whether a `null` can be passed with different behavior.
2. `workspaceConfig.route.passHostHeader`:  Default value is `false` so there doesnot appear to be a different is a Go boolean zero value (`false`) is sent or not.